scripts/apps/desks/controllers/DeskConfigController.ts

```diff
@@ -12,13 +14,28 @@ export function DeskConfigController($scope, $controller, notify, desks, WizardH
     };
     $scope.desk = {
         edit: null,
+        orig: null,
     };
 
     $scope.openDesk = function(step, desk) {
+        if (!desk) {
+            initializeDesk({}, step);
+            return;
+        }
+        desks.fetchDeskById(desk._id).then((_desk) => {
+            if (!_desk) {
```

Comment:
Move logging to `fetchDeskById` method and reject the promise if it's not found.